tests/Alcaeus/MongoDbAdapter/MongoClientTest.php

```diff
@@ -82,7 +82,8 @@ public function testWriteConcern()
 
     public function testListDBs()
     {
-        $this->getCollection()->insert(['foo' => 'bar']);
+        $document = ['foo' => 'bar'];
+        $this->getCollection()->insert($document);
```

Comment:
Gave that a shot, unfortunately it doesn't work since it would pass the return value of the assignment (which is a value, not a reference). Nice idea though!